lesson8-UnitTesting/FizzBuzz/src/FizzBuzz.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace FizzBuzz
+{
+    public class FizzBuzzClass
```

Comment:
Consider not using `Class` in the class name.

Reply:
done